python/lsst/meas/algorithms/installGaussianPsf.py

```diff
@@ -0,0 +1,145 @@
+#
+# LSST Data Management System
+# Copyright 2008-2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+import math
+
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.meas.algorithms import SingleGaussianPsf
+
+__all__ = ["InstallGaussianPsfConfig", "InstallGaussianPsfTask"]
+
+FwhmPerSigma = 2.0*math.sqrt(2.0*math.log(2.0))
+
+class InstallGaussianPsfConfig(pexConfig.Config):
+    """!Config for InstallGaussianPsfTask"""
+    fwhm = pexConfig.Field(
+        dtype = float,
+        default = 1.5 * FwhmPerSigma,
+        doc = "Estimated FWHM of simple Gaussian PSF model, in pixels. "
```

Comment:
It's not really an "estimated" FWHM.  It's the actual FWHM of the Gaussian PSF model.